crates/sol-macro-gen/src/sol_macro_gen.rs

```diff
@@ -125,11 +126,19 @@ impl MultiSolMacroGen {
 name = "{name}"
 version = "{version}"
 edition = "2021"
-
-[dependencies]
 "#
         );
 
+        if !description.is_empty() {
+            toml_contents.push_str(&format!("description = \"{}\"\n", description));
+        }
+
+        if !license.is_empty() {
+            toml_contents.push_str(&format!("license = \"{}\"\n", license));
+        }
```

Comment:
here we can add some nice mappings to properly format the value for this field:

https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields

so what we can do is add some builtin aliases that we check for, for example "mit,apache2" would map to "MIT OR Apache-2.0"

I think we can do this with a simple match here